sklearn/cross_validation.py

```diff
@@ -1794,10 +1794,14 @@ def train_test_split(*arrays, **options):
 
     Parameters
     ----------
-    *arrays : sequence of arrays or scipy.sparse matrices with same shape[0]
-        Python lists or tuples occurring in arrays are converted to 1D numpy
-        arrays.
-
+    *arrays : Sequence of arrays, scipy.sparse matrices or
+        Pandas DataFrames with same shape[0]. 
+        .. versionadded:: 0.6
```

Comment:
Sorry, I was a bit busy in the past few days. I will try and get this sorted out in the next couple of days.